default.nix

```diff
@@ -31,6 +32,12 @@ let
       kill $TAILPID
     '';
   });
+  # Enables building but not running of benchmarks when
+  # enableBenchmarks argument is supplied.
+  withBenchmarks = drv: if enableBenchmarks
+    then doBenchmark (appendConfigureFlag drv "--enable-benchmarks")
+    else drv;
```

Comment:
Can we call this `buildWithBenchmarks` to clarify it has to do with building the benchmarks rather than running them? When looking at the `withBenchmarks` call sites people may mistakenly think the latter.

Reply:
Fixed, thanks.